wrapper/CSharp/wolfSSL_CSharp/wolfSSL.cs

```diff
@@ -20,22 +20,180 @@
  */
 
 
+/* CE Not always reliably detected. Define our own WindowsCE as needed. */
+#if _WIN32_WCE || WINCE || PocketPC
+    /* WindowsCE should have been defined in the Project and user_settings.h  */
+    #if !WindowsCE
+        #define WindowsCE
+    #endif
+#endif
+
+
 using System;
 using System.Runtime.InteropServices;
 using System.Text;
-using System.Threading;
 using System.IO;
 using System.Net;
 using System.Net.Sockets;
 
 namespace wolfSSL.CSharp
 {
+    /********************************

```

Comment:
Shouldn't these be part of the `public class wolfssl`? I'd prefer use wolfssl.WOLFSSL_ALERT, etc in the calling code.

Reply:
> Shouldn't these be part of the public class wolfssl? 

Yes, it was at one point, but there was a scoping problem.  I'll see if I can move it back or document why not.